packages/annotorious/src/annotation/editors/point/PointEditor.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="ts">
+  import { boundsFromPoint, POINT_RADIUS } from '../../../model';
+  import type { Point, PointGeometry, Shape } from '../../../model';
+  import type { Transform } from '../../Transform';
+  import { Editor, Handle } from '..';
+
+  /** Props */
+  export let shape: Point;
+  export let computedStyle: string | undefined;
+  export let transform: Transform;
+  export let viewportScale: number = 1;
+
+  $: geom = shape.geometry;
+
+  const editor = (point: Shape, handle: string, delta: [number, number]) => {
+    const geom = point.geometry as PointGeometry;
+
+    const { x, y } = geom;
+    const [dx, dy] = delta;
+
+    const pointTranslated = {
+      x: x + dx,
+      y: y + dy,
+    };
+
+    const bounds = boundsFromPoint(pointTranslated.x, pointTranslated.y);
+
+    return {
+      ...point,
+      geometry: { ...pointTranslated, bounds },
+    };
+  };
+</script>
+
+<Editor {shape} {transform} {editor} on:change on:grab on:release let:grab>
+  <circle
+    class="a9s-outer"
+    style="fill:yellow;stroke:darkgoldenrod;"
```

Comment:
I don't know how to inform the user that the point is selected.

Reply:
I found:
![CleanShot 2024-09-12 at 16 41 11@2x](https://github.com/user-attachments/assets/cf3edf3a-fc15-4041-899a-da3dd29af6e4)